packages/react-i18n/src/i18n.ts

```diff
@@ -474,7 +474,7 @@ export class I18n {
     }).toLocaleLowerCase();
 
     if (isToday(date)) {
-      return time;
+      return `Today at ${time}`;
```

Comment:
The package should already do this for you. Maybe I can help you configure the package if you want.

Reply:
Yeah that'll be awesome. Thanks, I'll close the PR then